app/views/queries/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="container">
+  <%= form_with url: queries_perform_path(:flag),method: :get do |form| %>
+    <div class="form-check">
+      <%= form.radio_button :flag, "1",class: "form-check-input" %>
+      <%= form.label :flag, "Employees having age > 20 and < 40", class:"form-check-label" %>
+    </div>
```

Comment:
Create a hash or array for the filter values and loop it to create radio buttons